javascript/ql/src/Declarations/MixedStaticInstanceThisAccess.ql

```diff
@@ -10,15 +10,18 @@
 */
 import javascript
 
+MethodDefinition getASuperClassMethod(ClassDefinition base, string name) {
+  result = base.getMethod(name) or
+  result = getASuperClassMethod(base.getDirectSuperClassDefinition(), name)
```

Comment:
```suggestion
  result = getASuperClassMethod(base.getSuperClassDefinition(), name)
```

I don't think we need the "direct" anymore (Java, for example, calls the corresponding predicate `getASupertype`).